systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaRebalanceResource.java

```diff
@@ -48,10 +48,6 @@ public boolean waitForReadiness(KafkaRebalance resource) {
         return Crds.kafkaRebalanceOperation(ResourceManager.kubeClient().getClient());
     }
 
-    public static void replaceKafkaRebalanceResource(String resourceName, Consumer<KafkaRebalance> editor) {
-        ResourceManager.replaceCrdResource(KafkaRebalance.class, KafkaRebalanceList.class, resourceName, editor);
-    }
-
```

Comment:
Don't remove these types of methods...even if they are not used...For each our resource we must have class replace and namespace-wide replace